src/commons/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,38 @@
+import { CardInterface, ModalInterface } from "src/types";
+import Card from "../Card/Card";
+import styles from "./CardList.module.scss";
+
+interface CardListProps {
+  cardList?: CardInterface[];
+  onClick?: (m: ModalInterface) => void;
+  keyword?: string;
+}
+
+function CardList({ cardList, onClick, keyword = "" }: CardListProps) {
+  if (!cardList) {
+    return <div className={styles["no-card-list"]}> 잘못된 접근입니다.</div>;
+  }
+  if (keyword) {
+    cardList = cardList.filter((card) => {
+      return (
+        card.description?.includes(keyword) ||
+        card.url?.includes(keyword) ||
+        card.title?.includes(keyword)
+      );
+    });
+  }
+  if (cardList.length === 0) {
```

Comment:
```suggestion
  if (!cardList && cardList.length === 0) {
```
이렇게 하나 추가해주시면 더 좋을것같아요 :)